src/player.js

```diff
@@ -1842,6 +1852,7 @@ export default class Player extends FakeEventTarget {
    * @private
    */
   _handleAutoPlay(): void {
+    this._posterManager.show();
```

Comment:
It won't cause flickering in case of video autoplay?

Reply:
I didn't see a flicker

Reply:
In any case, this is logically not safe - if autoplay is set we shoudn't show a poster.
Need to move to where we know we decided to not show it.

Reply:
@Yuvalke try with throttling the browser